fsdp_config.yaml

```diff
@@ -0,0 +1,24 @@
+compute_environment: LOCAL_MACHINE
```

Comment:
is this meant to be committed to main? if so we should move to inside of the src dir (and will need to add to the `MANIFEST` file)

Reply:
This is an example config file, users would have to update it based on their # of GPUs and they may also want to change other settings. It probably makes most sense in an example directory with a README explaining how to change and use it, I'll move it